hasura/seeds/seedscript.ts

```diff
@@ -0,0 +1,152 @@
+import { writeFile } from "fs/promises";
+import { datatype, date, image, name, random } from "faker";
+// @ts-ignore
+const jsonSql = require("json-sql")();
```

Comment:
Maybe `import newJsonSql from "json-sql";` then `const jsonSql = newJsonSql();` would work without the `ts-ignore`?

Reply:
There is no type declaration file and no type package so typescript isn't happy, but I could use import anyway for sure